RUPG/APIManager.js

```diff
@@ -0,0 +1,55 @@
+class APIManager {
+    constructor() {
+        this.data = {}
+    }
+    peopleAPI() {
+        $.ajax({
+            url: 'https://randomuser.me/api/?results=7',
+            dataType: 'json',
+            success: (data) => {
+                let people = []
+                for (let i = 1; i < 7; i++) {
+                    const fname = data.results[i].name.first
+                    const lname = data.results[i].name.last
+                    const newPerson = { fname, lname }
+                    people.push(newPerson)
+                }
+                this.data.pp = data.results[0].picture.large
+                this.data.fname = data.results[0].name.first
+                this.data.lname = data.results[0].name.last
+                this.data.city = data.results[0].location.city
+                this.data.country = data.results[0].location.country
+                this.data.people = people
+            }
+        })
+    }
+    qouteAPI() {
+        $.ajax({
+            url: 'https://api.kanye.rest',
+            dataType: 'json',
+            success: (data) => {
+                this.data.quote = data.quote
+            }
+        })
+    }
+    pokemonAPI() {
+        const rInt = Math.floor(Math.random() * 894)
```

Comment:
you can call this `id` because the query from the pokemonAPI asked for an `${id}`